_posts/2020-04-28-history-of-bel.md

```diff
@@ -216,9 +260,9 @@ Before its dissolution, Dexter Pratt had already moved to UCSD and begun work on
 [NDEx](http://www.ndexbio.org/) project. Ex-CEO Keith Ellison and current CEO David de Graaf continued their careers
 in VC and entrepreneurship. Luckily, we have LinkedIn to figure this kind of stuff out.
 
-During its 15 years of operation, Selventa contracted an enormous amount of curation to generate BEL content. I'm
+During its 15 years of operation, Selventa performed an enormous amount of curation to generate BEL content. I'm
 not sure what the actual number but I've heard that it has millions of edges in it. After the closure of Selventa,
-it's not obvious what happened to the intellectual property of the company. Dexter Pratt asked through the
+ the intellectual property of the company was sold to Alexion. Dexter Pratt asked through the
```

Comment:
do you know any names of people who are using it right now? I'd be really keen to say hello